makefile.shade

```diff
@@ -1,91 +1,25 @@
-      // Does not handle directories under contentFiles/any/any.
-      var sourceDirectory = Path.Combine(extractToDirectory, "contentFiles", "any", "any");
-      if (Directory.Exists(sourceDirectory))
-      {
-          var targetDirectory = Path.Combine(extractToDirectory, "contentFiles", "any", "netstandard1.0");
-          if (!Directory.Exists(targetDirectory))
-          {
-              Directory.CreateDirectory(targetDirectory);
-          }
-
-          foreach (var file in Directory.GetFiles(sourceDirectory))
-          {
-              var fileName = Path.GetFileName(file);
-              var targetFile = Path.Combine(targetDirectory, fileName);
-              File.Copy(file, targetFile, overwrite: false);
-          }
-      }
-
-      // Update the .nuspec file.
-      var nuspecFile = Files.Include(Path.Combine(extractToDirectory, "*.nuspec")).First();
-      var nuspec = XDocument.Load(nuspecFile);
-      XNamespace nuspecNamespace = "http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd";
-      var package = nuspec.Element(nuspecNamespace + "package");
-      if (package == null)
-      {
-          // Try newer namespace.
-          nuspecNamespace = "http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd";
-          package = nuspec.Element(nuspecNamespace + "package");
-      }
-
-      var metadata = package.Element(nuspecNamespace + "metadata");
-      var files = new XElement(
-          nuspecNamespace + "files",
-          new XAttribute("include", "**/*.resx"),
-          new XAttribute("buildAction", "EmbeddedResource"));
-      var contentFiles = new XElement(nuspecNamespace + "contentFiles", files);
-
-      metadata.Add(contentFiles);
-      nuspec.Save(nuspecFile);
-
-      // Repack.
-      var nugetExePath = Environment.GetEnvironmentVariable("KOREBUILD_NUGET_EXE");
-      if (string.IsNullOrEmpty(nugetExePath))
-      {
-          nugetExePath = Path.Combine(BASE_DIR_LOCAL, ".build", "nuget.exe");
-      }
-      ExecClr(nugetExePath, "pack " + nuspecFile + " -OutputDirectory " + BUILD_DIR_LOCAL);
 
-      try
-      {
-          // Delete temporary directory we used to repack.
-          Directory.Delete(extractToDirectory, true);
-      }
-      catch
-      {
-          // Don't care if we couldn't delete the temp directory.
-      }
-   }
\ No newline at end of file
+#pack-sources target='compile'
+    @{
+        foreach (var dir in Directory.GetDirectories(Directory.GetCurrentDirectory(), "src/*.Sources"))
+        {
+            var nugetPath = E("KOREBUILD_NUGET_EXE") ?? ".build/NuGet.exe";
+            var outputDir = "artifacts/build/";
+            var packageVersion = "1.2.0-" + E("DOTNET_BUILD_VERSION");
+            NugetPack(nugetPath,
+                "sources.nuspec",
+                outputDir,
+                packageVersion,
+                "-BasePath " + dir + " -Properties id=" + Path.GetFileName(dir));
```

Comment:
Need to escape `dir` here in case it has spaces? Can it go in quotes or something?